### PR TITLE
Mobile location component styling 

### PR DESCRIPTION
**Pull request by alextaing**
Changed location bias styling on mobile such that on mobile devices the location, attribution, and button show vertically. See this [slack thread](https://yext.slack.com/archives/C032CKFARGS/p1657121613510669) for more information.

J=SLAP-2205
TEST=manual

Checked using dev tools on Chrome that expected behavior occurred.

### CODE REVIEW

**Review by oshi97 — APPROVED** [review]

File: src/components/LocationBias.tsx [hunk 1]
```diff
@@ -11,15 +11,15 @@ import LoadingIndicator from '../icons/LoadingIndicator';
...
+  locationBiasContainer?: string,
   location?: string,
   source?: string,
   button?: string,
   loadingIndicatorContainer?: string
 }
 
 const builtInCssClasses: Readonly<LocationBiasCssClasses> = {
-  container: 'text-sm text-neutral text-center flex justify-center items-center',
+  locationBiasContainer: 'text-sm text-neutral text-center justify-center items-center flex flex-col lg:flex-row',
```
Comment (nmanu1) [inline src/components/LocationBias.tsx:22]: would `md:flex-row` work here instead of `lg:flex-row`?

Comment (alextaing) [reply src/components/LocationBias.tsx:22]: if you use md, there's this small awkward range where it looks scrunched like the original issue.   lg seems to work well for names with reasonable length
image

**Review by nmanu1 — COMMENTED** [review]
It looks like the Percy snapshots for when location bias is loading has changed, even on large screens. The loading indicator isn't quite centered vertically anymore

File: src/components/LocationBias.tsx (hunk 1 above)
Comment (nmanu1) [reply src/components/LocationBias.tsx:22]: ok, `lg` is fine then!

File: src/components/LocationBias.tsx [hunk 2]
```diff
@@ -79,26 +79,31 @@ export function LocationBias({
...
-    <div className={cssClasses.container}>
-      {isFetchingLocation && <div className={cssClasses.loadingIndicatorContainer}/>}
+    <div className={cssClasses.locationBiasContainer}>
+      {screenIsLarge && isFetchingLocation && <div className={cssClasses.loadingIndicatorContainer}/>}
       <span className={cssClasses.location}>
         {locationBias.displayName}
       </span>
       <span className={cssClasses.source}>
         {attributionMessage}
       </span>
```
Comment (yen-tt) [inline src/components/LocationBias.tsx:92]: in percy, I see that there's a new space gap between `(base on your device) -` and `update your location`, can we remove that?
Screen Shot 2022-07-07 at 10 07 06 AM
Screen Shot 2022-07-07 at 10 07 08 AM

**Pull request closed without merging**